lib/decidim/decidim_awesome/version.rb

```diff
@@ -4,6 +4,6 @@ module Decidim
   # This holds the decidim-decidim_awesome version.
   module DecidimAwesome
     VERSION = "0.7.2"
-    COMPAT_DECIDIM_VERSION = [">= 0.23.0", "< 0.25"].freeze
+    COMPAT_DECIDIM_VERSION = [">= 0.23.0", "< 0.26"].freeze
```

Comment:
```suggestion
    COMPAT_DECIDIM_VERSION = [">= 0.25.0", "< 0.26"].freeze
```
I think this should be the version supported for the next version of awesome. Maintaining sprockets compatibility seems to much for what's is worth.